nautobot/docs/release-notes/version-1.0.md

```diff
@@ -177,6 +177,7 @@ The "Related Devices" table has been removed from the detailed Device view.
 ### Added
 
 - [#430](https://github.com/nautobot/nautobot/pull/430) - GraphQL `ip_addresses` now includes an `assigned_object` field
+- [#456](https://github.com/nautobot/nautobot/issues/456) - Added mkdocs-include-markdown-plugin
```

Comment:
Sorry to nitpick, but let's keep this listing in numerical order please (456 after 451). Thanks!